Mistica/Source/Components/Callout/CalloutConfiguration.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  CalloutConfiguration.swift
+//
+//  Made with ❤️ by Novum
+//
+//  Copyright © Telefonica. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public struct CalloutConfiguration {
+    static let emptyConfiguration = CalloutConfiguration(withDescription: "Empty configuration", asset: .none, title: nil, actions: nil, canClose: true)
+
+    public enum CalloutActions {
+        case primary(CalloutButton)
+        case primaryAndLink(primary: CalloutButton, link: CalloutLinkButton)
+        case primaryAndSecondary(primary: CalloutButton, secondary: CalloutButton)
+        case secondary(CalloutButton)
+        case secondaryAndLink(secondary: CalloutButton, link: CalloutLinkButton)
+        case link(CalloutLinkButton)
+        case empty
+    }
+
+    @frozen
+    public enum CalloutAssetType: Equatable {
+        case none
+        case smallIcon(UIImage)
+    }
+
+    let asset: CalloutAssetType
+    let title: String?
+    let descriptionTitle: String
```

Comment:
This should be called:

```suggestion
    let description: String
```